### DISCUSSION TITLE
developer-case-studies/transcriber-ai-metadata/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#398] [Website Discussions]
# developer-case-studies/transcriber-ai-metadata/

This is a story that starts not so long ago, about what I call the hardest job of all after being a dad: being a good husband...

https://fcp.cafe/developer-case-studies/transcriber-ai-metadata/

**Comment by clementunes** [comment]
Great article. Love these behind the scenes kinds of narrative. Checking out your plugin and podcast. Thanks for sharing!!

**Reply by shylock74** [reply]
Thank you ❤️ 🙂
